cmd/fleet/serve.go

```diff
@@ -1090,7 +1090,15 @@ the way that the Fleet server works.
 					frontendHandler = service.RedirectSetupToLogin(svc, logger, frontendHandler, config.Server.URLPrefix)
 				}
 
-				endUserEnrollOTAHandler = service.ServeEndUserEnrollOTA(svc, config.Server.URLPrefix, logger)
+				// TODO: need a mechanism to check if android feature is enabled
+				endUserEnrollOTAHandler = service.ServeEndUserEnrollOTA(
```

Comment:
You could check the environment variable ([example](https://github.com/fleetdm/fleet/blob/49f6616a582e42458a3485a304a9e05dc377f24e/cmd/fleet/serve.go#L199-L200)).

Reply:
thank you! will make this change